transfers/file-transfers.md

```diff
@@ -99,19 +110,35 @@ Or
 
 * /ACQ_1234_123456
   * /metadata
-  * /objects
+  * /objects-->
+
+### Transfer files in office with transfer_rsync.py
+
+{: .important}
+Files that have been updated by the donor within the past 30 days should be quarantined for 30 days to ensure that all virus definitions are up to date.
+
+Files are transferred from digital carriers in office using the [transfer_rsync script](https://nypl.github.io/digarch/tools/working-scripts.html#transfer_rsyncpy){:target="_blank"}. This script uses rsync, a command utility used for copying and syncing file locations, to initiate file transfer then packages the files in our digital archives package structure. 
 
-## Transfer files from media object
+More information about rsync can be found on the [rsync documentation page](https://linux.die.net/man/1/rsync) and [installation instructions](../software#rsync) can be found in our software section. More information about [transfer_rsync.py](https://nypl.github.io/digarch/tools/working-scripts.html#transfer_rsyncpy){:target="_blank"} can be found in the [working scripts](https://nypl.github.io/digarch/tools/working-scripts){:target="_blank"} section of our documentation.
 
-Files that have been updated by the donor within the past 30
-days should be quarantined for 30 days to ensure that
-all virus definitions are up to date.
+* Use a write-blocker to connect the drive to the computer. See [Using Tableau Write Blockers](/digarch/transfers/using-tableaus.html){:target="_blank"} for instructions.
+
+* Open terminal on Mac or WSL on Windows
+
+* Run transfer_rsync.py by entering the following command with the appropriate paths: ```python3 transfer_rsync.py –-carrierid (M#### or ACQ_####_######) -–payload path/to/files/directory -–dest path/to/destination/directory```
```

Comment:
```suggestion
* Run transfer_rsync.py by entering the following command with the appropriate paths: ```python3 transfer_rsync.py –-carrierid (M#### or ACQ_####_######) -–payload path/to/files/directory -–dest path/to/destination/directory```
```
* Run transfer_rsync.py by entering the following command with the appropriate paths:  ```python3 transfer_rsync.py –-carrierid (M#### or ACQ_####_######) -–payload path/to/files/directory -–dest path/to/destination/directory

Reply:
make sure python3 starts on a new line